### PR TITLE
Augmented node template docs

### PR DESCRIPTION
**Pull request by danforbes**

### CODE REVIEW

**Comment by danforbes** [comment]
bot merge

**Comment by danforbes** [comment]
bot merge :pray:

**Comment by danforbes** [comment]
bot merge

**Comment by bkchr** [comment]
bot merge

File: bin/node-template/README.md [hunk 1]
```diff
@@ -92,6 +93,91 @@ cargo run -- \
...
+```shell
+./target/release/node-template --help
+```
+
+### Runtime
+
+The Substrate project in this repository uses the
+[FRAME](https://substrate.dev/docs/en/knowledgebase/runtime/frame) framework to construct a
+blockchain runtime. This means that the code that defines the runtime is separated into modules
+called "pallets". At the heart of FRAME is a helpful
```
Comment (joepetrowski) [inline bin/node-template/README.md:146]:
```suggestion
blockchain runtime. FRAME allows you to declare domain-specific logic in modules called "pallets".
At the heart of FRAME is a helpful
```

**Pull request merged**